src/core/primitives.js

```diff
@@ -33,6 +33,12 @@ const Name = (function NameClosure() {
       return nameValue ? nameValue : (nameCache[name] = new Name(name));
     }
 
+    static get empty() {
+      // eslint-disable-next-line no-restricted-syntax
+      const emptyName = new Name({ empty: true });
+      return shadow(this, "empty", emptyName);
+    }
```

Comment:
The other possible solution, which I suppose could be seen as overall preferable, would be just remove [checks such as this one](https://github.com/mozilla/pdf.js/blob/9441245320ba80b91cc9e94da5205a394a165f7b/src/core/evaluator.js#L1571-L1573) and thus allow empty `Name`s in the "normal" way!?